Xamarin.Forms.Core/ColumnDefinitionCollectionTypeConverter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	[Xaml.TypeConversion(typeof(ColumnDefinitionCollection))]
+	public class ColumnDefinitionCollectionTypeConverter : TypeConverter
+	{
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (value != null) {
+				var lengths = value.Split(',');
```

Comment:
To follow common xaml, both spaces and commas can be used. See the original spec that the issue linked to .
Also see the Thickness Converter.